Robust/Repositories/CartRepository.cs

```diff
@@ -142,5 +142,22 @@ public ObservableCollection<CartItem> GetAll(string username, string password)
 
             return cartItems;
         }
+        
+        public bool Update(CartItem cartItem, int quantity, int customerId = 1)
```

Comment:
Det her vil være et problem pga vi stoler på brugeren for meget her kan de sige "customerId 2" også siger vi bare ok.

Vi skal derfor bruge vores API jeg tilføjede i sidste PR også send username og password til vores repository for så at få customerId udfra deres logind oplysninger